lib.php

```diff
@@ -70,6 +70,10 @@
 define('F2F_CAL_COURSE', 1);
 define('F2F_CAL_SITE',   2);
 
+// Signup setting constants.
```

Comment:
This violates the naming conventions on constant names... I know there are other existing constants that don't meet the naming guidelines (and this follows that format) but it would be preferable to not add more.....

Reply:
Changed them to `MOD_FACETOFACE_SIGNUP_...`